CMakeLists.txt

```diff
@@ -622,9 +622,8 @@ if(NOT CONFIG_ARCH_SIM)
 
   # TODO: nostart/nodefault not applicable to nuttx toolchain
   target_link_libraries(
-    nuttx
-    PRIVATE ${NUTTX_EXTRA_FLAGS} -Wl,--script=${ldscript} -Wl,--start-group
-            ${nuttx_libs} ${nuttx_extra_libs} -Wl,--end-group)
+    nuttx PRIVATE -Wl,--script=${ldscript} -Wl,--start-group ${nuttx_libs}
```

Comment:
Why ${NUTTX_EXTRA_FLAGS} was removed?

Reply:
> Why ${NUTTX_EXTRA_FLAGS} was removed?

This NUTTX_EXTRA_FLAGS is a compile option set in the configure stage. I think this is redundant when linking.